NOTES.UNIX

```diff
@@ -92,20 +92,26 @@
         '-Wl,--enable-new-dtags,-rpath,$(LIBRPATH)'
 
  It might be worth noting that some/most ELF systems implement support
- for relative runtime search path, relative to directory containing
- the executable object, by interpreting $ORIGIN along with some other
- internal variables. Consult your system documentation.
+ for runtime search path relative to the directory containing current
+ executable, by interpreting $ORIGIN along with some other internal
+ variables. Consult your system documentation.
 
  Linking your application
  ------------------------
 
- Above is the story of building and installing OpenSSL shared libraries
- and application. But third-party applications dynamically linked with
- OpenSSL (or any other) shared library face exactly same problem with
- non-default locations. Naturally abovementioned OpenSSL config options
- do not necessarily have bearing on linking of the target application.
- "Not necessarily" means that under some circumstances it would be
- sufficient to link with OpenSSL shared library "naturally", i.e. with
- -L/whatever/path -lssl -lcrypto. But there also are cases when you'd
- have to specify runtime search path yourself. Consult your system
- documentation and use above section as inspiration...
+ Third-party applications dynamically linked with OpenSSL (or any other)
+ shared library face exactly same problem with non-default locations.
+ Abovementioned OpenSSL config options might or might not have bearing
+ on linking of the target application. "Might" means that under some
+ circumstances it would be sufficient to link with OpenSSL shared library
+ "naturally", i.e. with -L/whatever/path -lssl -lcrypto. But there also
+ are cases when you'd have to explicitly specify runtime search path
```

Comment:
s/But the also are cases/But there are also cases/

Reply:
It says "but there are also cases"... Or what am I missing?

Reply:
Nope, it says "But there also are cases".  "also" and "are" need to trade places.

Reply:
Ah! I was hooked on "the" vs. "there"...